examples/security-insights-sample.yml

```diff
@@ -66,18 +75,18 @@ security-artifacts:
       sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt
       mollit anim id est laborum
   other-artifacts:
-    - artifact-name: example-artifact
-      artifact-created: true
-      evidence-url:
-      - https://foo.com/artifact.html
-      comment: |
-        Lorem ipsum dolor sit amet, consectetur adipisci elit,
-        sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.
-        Ut enim ad minim veniam, quis nostrum exercitationem ullamco laboriosam,
-        nisi ut aliquid ex ea commodi consequatur. Duis aute irure reprehenderit
-        in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
-        sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt
-        mollit anim id est laborum
+    artifact-name: example-artifact
```

Comment:
Is this correct? I believe this should still be a list item.

Reply:
@eddie-knight here is the fix 🥳 https://github.com/ossf/security-insights-spec/pull/48